build/Build.cs

```diff
@@ -139,9 +142,19 @@ partial class Build : NukeBuild
       .DependsOn(Tests)
       .Executes(() =>
       {
-          //Since this is about a new release, `RunChangeLog` need to be executed to update the ChangeLog.md with the new version
-          //from which LatestVersion will be parsed
           var version = GitVersion.SemVer;
+          var branchName = GitVersion.BranchName;
+
+          if(branchName.Equals(ReleaseBranch, StringComparison.OrdinalIgnoreCase) 
+          && !GitVersion.MajorMinorPatch.Equals(LatestVersion.Version.ToString()))
```

Comment:
Although `GitVersion` handles the `SemVer` when making final releases, here I want to force user to update the `ChangeLog.md` in case it skipped the mind. And the way it is updated is by executing the `RunChangeLog` command `build.cmd runchangelog`. Doing so will create a new version section and move everything from `vNext` into it.